src/internal/coredb/projects.go

```diff
@@ -3,19 +3,29 @@ package coredb
 import (
 	"context"
 	"fmt"
+	"io"
+	"time"
+
 	"github.com/gogo/protobuf/types"
 	"github.com/jmoiron/sqlx"
+
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
 	"github.com/pachyderm/pachyderm/v2/src/pfs"
-	"io"
-	"time"
 )
 
 type ProjectIterator struct {
 	*sqlx.Rows
 }
 
+// ListProjectOption contains optional parameters that support pagination.
+// PageSize and PageNum are converted as-is into an SQL LIMIT and OFFSET.
```

Comment:
Hmm I interpreted "as-is" as directly converting PageNumber to OFFSET and PageSize to LIMIT, where  in fact `OFFSET = PageNum * PageSize`